DataManagementSystem/scripts/dirac-dms-replicate-and-register-request.py

```diff
@@ -33,18 +32,16 @@ def getLFNList( LFNs ):
   args = Script.getPositionalArgs()
 
   requestName = None
-  LFNs = None
   targetSEs = None
   if len( args ) < 3:
     Script.showHelp()
-    DIRAC.exit( 0 )
-  else:
-    requestName = args[0]
-    LFNs = args[1]
-    targetSEs = list( set( [ targetSE.strip() for targetSE in args[2:] ] ) )
+    DIRAC.exit( 1 )
+
+  requestName = args[0]
+  lfnList = getLFNList( args[1] )
+  targetSEs = list( set( [ se for targetSE in args[2:] for se in targetSE.split( ',' ) ] ) )
```

Comment:
with this loop, are you trying to cover the case of someone doing:
...... /my/lfn se1,se2 se3 
?
If yes, I think we should not.
If not, then list(set(args[2].split(',') )) should be enough no ?